egs2/librilight/lm1/run.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+. ./db.sh
+
+lm_train_text=data/lm_train.txt
+lm_dev_text=data/lm_valid.txt
+lm_test_text=data/lm_test.txt
+
+lm_config=conf/tuning/train_lm_rnn_unit1024_nlayers3_dropout0.2_epoch30.yaml
+
+./asr.sh \
+    --stage 6 \
+    --stop_stage 8 \
```

Comment:
Are they OK?
Can you really skip the data preparation part?
Can you double-check it?

Reply:
Can you add some instructions on why we only use stages 6-8?
How about adding a comment to check `README.md` first?